ci/gpu/build.sh

```diff
@@ -81,14 +81,29 @@ logger "Adding ${CONDA_PREFIX}/lib to LD_LIBRARY_PATH"
 export LD_LIBRARY_PATH_CACHED=$LD_LIBRARY_PATH
 export LD_LIBRARY_PATH=$CONDA_PREFIX/lib:$LD_LIBRARY_PATH
 
-logger "Build libcuml..."
+logger "Build libcuml, cuml, prims and bench targets..."
 $WORKSPACE/build.sh clean libcuml cuml prims bench -v
 
 logger "Resetting LD_LIBRARY_PATH..."
 
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH_CACHED
 export LD_LIBRARY_PATH_CACHED=""
 
+logger "Build treelite for GPU testing..."
+# Buildint treelite Python for testing is temporary while there is a pip/conda
+# treelite package
+
+cd $WORKSPACE/cpp/build/treelite/src/treelite
+mkdir build
+cd build
+cmake ..
```

Comment:
Shame we have to build this twice (since it's in the real cmake too to link with it, right?) but I think it's pretty quick, so not a huge concern.

Reply:
We actually don't build the dynamic treelite.so (just the static treelite.a) so there was no way of reusing those artifacts without rebuilding :(. But it should be very temporary, once there is a pip package all of this should be a simple pip command (or even better part of the conda recipe/command)